### PR TITLE
Issue #1163 - Convert ReverseResistorColours to use templates

### PR DESCRIPTION
**Pull request by Mailkov**
IA Page: https://duck.co/ia/view/reverse_resistor_colours

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@Mailkov this LGTM, the only thing that bothers me is the colon on the subtitle
image

File: lib/DDG/Goodie/ReverseResistorColours.pm [hunk 1]
```diff
@@ -17,63 +17,83 @@ name 'ReverseResistorColours';
...
+        $resistance = $resistance / 1000000;
+        $append = 'M';
+    }
+    if ($resistance > 1000000000) {
+        $resistance = $resistance / 1000000000;
+        $append = "G";
+    }
+    #U+2126 is the ohm symbol, U+00B1 is the plus-minus sign.
+    my $title = "$resistance $append\x{2126} \x{00B1} $marginE\%";
+    my $subtitle = "Resistance of $_ resistor:";
```
Comment (moollaza) [inline lib/DDG/Goodie/ReverseResistorColours.pm]: @mintsoft agreed!

@Mailkov let's remove the `:`

**Comment by Mailkov** [comment]
@mintsoft @moollaza I deleted the colon on the subtitle!

**Comment by GuiltyDolphin** [comment]
@Mailkov If you integrate changes from master those failing tests should go away (Conversions has been updated).

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**